Cargo.lock

```diff
@@ -14,9 +14,9 @@ dependencies = [
 
 [[package]]
 name = "addr2line"
-version = "0.21.0"
+version = "0.22.0"
```

Comment:
Why was this lockfile update done?

Reply:
`cargo generate-lockfile` did this automatically.

Reply:
Why was that run?

Reply:
An update on the master branch caused a CONFLICT in the lockfile in this PR, so I recreated it.

Reply:
Please don't throwaway existing lockfiles.  If nothing else, this cases a lot of unnecessary churn to a PR.  A way to resolve conflicts like this is `cargo checkout master -- Cargo.lock && cargo check`.

Reply:
done

Reply:
ok